docs/development_testing.md

```diff
@@ -17,6 +17,12 @@ To run our linters, on the full code base, run the following command:
 pre-commit run --all-files
 ```
 
+If you want to run the full test suite, you need to install all dependencies. Activate the virtual environment and run the command:
```

Comment:
maybe move this up to L12 and start with something like "If not using the VSCode devcontainer.....", since the sentence on L12 mentions how to run the tests using devcontainer, and this is an alternative to it

Reply:
Apparently it wasn't clear that this is a step required when using devcontainers.
I have moved it to the getting started section of devcontainer instead.
Hopefully this is more clear?

Reply:
Sorry I'm going to counter review this.

Most people really do not need this. This should suggestion and change should be undone.

Reply:
I have reverted the change to the original place.
But reworded it a bit to hopefully make it more clear.

Reply:
I agree with @frenck  that this should not be on the devcontainer section. My suggestion was to place it after that sentence on L12 with a different wording.

L12 states: `This can be achieved by using the VScode devcontainer and the corresponding task.`, because the vscode tasks install all the dependencies automatically, thus it is not needed if people use the devcontainer and the vscode tasks. But for someone not using vscode, this is needed.

Reply:
@abmantis Is this phrasing better?

To run the full test suite, more dependencies are required than what is setup in the devcontainer by default. 
If you are running the vscode devcontainer, you can execute the 'Install all test requirements' task for this.
As an alternative you can activate the virtual environment and run the command: